TurkishFilter/sections/css_extended.txt

```diff
@@ -9,6 +9,7 @@ vizyonfilmizle.tv#$?#.fancybox-overlay-fixed[-ext-has="> div"] { display: none!i
 vizyonfilmizle.tv#$?#.fancybox-lock body[-ext-has="> div"] { overflow: visible!important; }
 vizyonfilmizle.tv#$?#html[-ext-has="> body"] { overflow: visible!important; }
 !
+donghuatr.com#?#.widget_text > .releases > h3:contains(/^Reklam$/):upward(2)
```

Comment:
`donghuatr.com#$?##sidebar > div.widget_text:has(> div > h3:contains(Reklam))` is faster.

Reply:
On simple sites the difference is not noticeable, but in the case of sites with a large number of dynamically created elements, it can slow down.